src/jltypes.c

```diff
@@ -2269,7 +2269,7 @@ void jl_init_types(void) JL_GC_DISABLED
                             "pure"),
                         jl_svec(18,
                             jl_array_any_type,
-                            jl_any_type,
+                            jl_array_int32_type,
```

Comment:
There is a line in method.c that sets this to `jl_nothing`. Should change that to NULL so nobody gets confused. There is also an assertion `src.codelocs !== nothing` in show.jl that can be removed.